packages/markdown/src/plugins/assetsPlugin/assetsPlugin.ts

```diff
@@ -13,6 +13,17 @@ export interface AssetsPluginOptions {
    * Prefix to add to relative assets links
    */
   relativePathPrefix?: string
+
+  /**
+   * The strictness of the determination of relative paths
+   *
+   * - strict: startWith `./` or `../` -> relative path
+   * - half: no startWith `/` or `@` or `<protocol>` (`https?:` `data:`) -> relative path
+   *   At this point, only aliases beginning with `@` are allowed to replace
+   *   relative paths not beginning with `.`
+   * - no: no startWith `/` or `<protocol>` -> relative path
+   */
+  restrictRelativePath?: 'half' | 'no' | 'strict'
```

Comment:
The option name and values are confusing.

For me, I would prefer `aliasSupport` with `'@-prefix' | boolean`